text/0001-rfc_process.md

```diff
@@ -0,0 +1,244 @@
+    convincing motivation, demonstrate understanding of the impact of the
+    design, or are disingenuous about the drawbacks or alternatives tend to be
+    poorly-received.
+  - Submit a pull request. As a pull request the RFC will receive design
+    feedback from the larger community, and the author should be prepared to
+    revise it in response.
+  - Each pull request will be reviewed by the core team and assigned to a team
+    member, who will take responsibility for guiding the RFC
+  - Build consensus and integrate feedback. RFCs that have broad support are
+    much more likely to make progress than those that don't receive any
+    comments. Feel free to reach out to the RFC assignee in particular to get
+    help identifying stakeholders and obstacles.
+  - The core team will discuss the RFC pull request, as much as possible in the
+    comment thread of the pull request itself. Offline discussion will be
+    summarized on the pull request comment thread.
+  - RFCs rarely go through this process unchanged, especially as alternatives
+    and drawbacks are shown. You can make edits, big and small, to the RFC to
+    clarify or change the design, but make changes as new commits to the pull
+    request, and leave a comment on the pull request explaining your changes.
+    Specifically, do not squash or rebase commits after they are visible on the
+    pull request.
+  - If the proposal is submitted by a core team member, it can be merged
+    by at least 2 other core team members approving the RFC, otherwise a member 
+    of the core team will propose a "motion for final comment period" (FCP), 
+    along with a *disposition* for the RFC (merge, close, or postpone).
+    - This step is taken when enough of the tradeoffs have been discussed that
+    the core team is in a position to make a decision. That does not require
+    consensus amongst all participants in the RFC thread (which is usually
+    impossible). However, the argument supporting the disposition on the RFC
+    needs to have already been clearly articulated, and there should not be a
+    strong consensus *against* that position outside of the core team. Team
+    members use their best judgment in taking this step, and the FCP itself
+    ensures there is ample time and notification for stakeholders to push back
+    if it is made prematurely.
+    - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+      a *summary comment* trying to lay out the current state of the discussion
+      and major tradeoffs/points of disagreement.
+    - Before actually entering FCP, *all* members of the core team must sign off;
+    this is often the point at which many core team members first review the RFC
+    in full depth.
+  - The FCP lasts ten calendar days, so that it is open for at least 5 business
+    days. It is also advertised widely,
+    e.g. in [This Week in Qri](https://this-week-in-rust.org/). This way all
```

Comment:
This is currently linking to https://this-week-in-rust.org/

Do we have a This Week in Qri? If not, we should have it before this gets merged.

Reply:
we do not, will remove 😄